docs/source/Freva.rst

```diff
@@ -50,7 +50,7 @@ to get an overview over existing plugins and the documentation of each plugins.
 
 This specific plugin has created the following output:
 
-.. image:: _static/animator_output.gif
+.. image:: _static/animator_output.mp4
```

Comment:
Is that an mp4 now? If so then we should make sure that it's still a gif. Because mp4 is not an image - as it's supposed to be.

Reply:
yes, animator produces mp4 and in the api/plugin.py

```
        if "type" not in metadata:
            ext = os.path.splitext(file_path)
            if ext:
                ext = ext[-1].lower()
                if ext in ".jpg .jpeg .png .gif .mp4 .mov".split():
                    metadata["type"] = "plot"
                    metadata["todo"] = "copy"

                if ext in ".tif .svg .ps .eps".split():
                    metadata["type"] = "plot"
                    metadata["todo"] = "con
```
I think it only copies with .gif .mp4 etc. and does not make any transformation.

Reply:
If I remember correctly, it's not at all. I added an animated gif that will be there regardless. There is a file called animator_output.gif `docs/source/_static/` and this is (or should be) just taken. So it pretends as this gif was created by the animator. In fact, it was, but a really long time ago.

Reply:
Ah, okey. The .gitignore was ignoring the .gif all this time (since many months ago, I guess).  The gif output weights 105 KB

I can take out that line from gitiginore and push it again with thte output content